README.md

```diff
@@ -67,86 +82,58 @@ The table below outlines what version of Nexus Repository the plugin was built a
 | v1.0.2         | >=3.14.0-04              |
 | v1.0.3         | >=3.15.2-01              |
 | v1.0.4         | >=3.18.0-01              |
-
-If a new version of Nexus Repository is released and the plugin needs changes, a new release will be made, and this
-table will be updated to indicate which version of Nexus Repository it will function against. This is done on a time 
-available basis, as this is community supported. If you see a new version of Nexus Repository, go ahead and update the
-plugin and send us a PR after testing it out!
-
+| v1.1.0 In product | >=3.20.0              |
 All released versions can be found [here](https://github.com/sonatype-nexus-community/nexus-repository-r/releases).
 
 ## Installing the plugin
+Starting from Nexus Repository Manager 3.20+ `R` format is already included. So there are no need to install it.  But if you want to reinstall the plugin with your improvements then following instructions will be useful. <br> <b>Note:</b> Using an unofficial version of the plugin is not supported by the Sonatype Support team.
```

Comment:
Minor: Because you say "Starting from" the + (in 3.20+) isn't necessary.  Alternatively you could say "In Nexus Repository Manager 3.20+, the `R`...".  Basically you have a redundancy.

Tangential, "are no need" should be "is no need".

Q: While the Note is true, is there something you're trying to say about "unofficial"?  I'm trying to figure out why this is here rather than elsewhere.

Reply:
37c8407965e8aedde7842d938c2762ba90b3649c